### PR TITLE
Cedric Docs update

### PR DESCRIPTION
**Pull request by nift4**

### CODE REVIEW

File: supplementary/devices/cedric.rst [hunk 1]
```diff
@@ -3,8 +3,11 @@ Cedric - Motorola Moto G5
 =========================
 
 The Moto G5 is a mid-range phone, with a few variants.
-
-The XT1676 variant is special in that it has 3 GB of RAM.
```
Comment (doniks) [inline supplementary/devices/cedric.rst:7]: Should we keep the detail that 1676 has 3gb somewhere?

File: supplementary/devices/cedric.rst [hunk 2]
```diff
@@ -49,13 +58,23 @@ Distributions
...
      - `android_device_motorola_cedric/cm-14.1 <https://github.com/Wzedlare/android_device_motorola_cedric/tree/cm-14.1>`_
      - `android_kernel_motorola_msm8937/cm-14.1 <https://github.com/Wzedlare/android_kernel_motorola_msm8937/tree/cm-14.1>`_ based on v3.18.49
-     - The vast majority of things
-     - ?
-   * - Unofficial LineageOS 15.1 (GitHub User: Wzedlare)
-     - `android_device_motorola_cedric/lineage-15.1 <https://github.com/Wzedlare/android_device_motorola_cedric/tree/lineage-15.1>`_
-     - `android_kernel_motorola_msm8937/lineage-15.1 <https://github.com/Wzedlare/android_kernel_motorola_msm8937/tree/lineage-15.1>`_ based on v3.18.100
-     - ?
-     - ?
+     - Eveerything
```
Comment (doniks) [inline supplementary/devices/cedric.rst:61]: Typo

File: supplementary/devices/cedric.rst [hunk 3]
```diff
@@ -49,13 +58,23 @@ Distributions
...
+   * - Unofficial LineageOS 15.1 (GitHub User: Sohamlad7)
+     - `android_device_motorola_cedric/lineage-15.1 <https://github.com/Sohamlad7/android_device_motorola_cedric/tree/lineage-15.1>`_
+     - `android_kernel_motorola_msm8937/lineage-15.1 <https://github.com/Sohamlad7/android_kernel_motorola_msm8937/tree/lineage-15.1>`_ based on v3.18.100
+     - Almost everything
+     - USB tethering (can be fixed tho)
+   * - Unoffical LineageOS 16.0 (GitHub User: Sohamlad7)
+     - `android_device_motorola_cedric/lineage-16.0 <https://github.com/Sohamlad7/android_device_motorola_cedric/tree/lineage-16.0>`_
+     - `android_kernel_motorola_msm8937/lineage-16.0 <https://github.com/Sohamlad7/android_kernel_motorola_msm8937/tree/lineage-16.0>`_ based on the newest 3.18
+     - Almost everything
+     - Audio for Wfd & Camera problems
```
Comment (doniks) [inline supplementary/devices/cedric.rst:72]: What is Wfd?

File: supplementary/devices/cedric.rst (hunk 1 above)
Comment (nift4) [reply supplementary/devices/cedric.rst:7]: I have an XT1676 with 2GB.

File: supplementary/devices/cedric.rst (hunk 3 above)
Comment (nift4) [reply supplementary/devices/cedric.rst:72]: Chromecast & Miracast.
Wfd is an abbreviation for Wireless Display.

File: supplementary/devices/cedric.rst (hunk 2 above)
Comment (nift4) [reply supplementary/devices/cedric.rst]: Fixed

**Pull request merged**